var/spack/repos/duplicates.test/packages/py-shapely/package.py

```diff
@@ -15,4 +15,4 @@ class PyShapely(Package):
 
     extends("python")
     depends_on("py-numpy", type=("build", "link", "run"))
-    depends_on("py-setuptools@=60", type="build")
+    depends_on("py-setuptools@=60", type=("build", "run"))
```

Comment:
I think we should avoid this change:
```suggestion
    depends_on("py-setuptools@=60", type="build")
```
Without it, we model the use case in the issue correctly. Besides, with the change, all unit tests pass also on develop (without, instead, 2 tests would fail on develop).